lib/codelog/cli.rb

```diff
@@ -19,16 +19,20 @@ def new(name = 'change')
     end
 
     desc 'release [VERSION] <RELEASE_DATE>', 'Generate new release updating changelog'
+    method_option :preview, desc: 'Prints the preview of the specified version release',
```

Comment:
Did you consider creating another command for the preview?

Reply:
@rmohashi  Actually yes, i just thought that using a optional argument would be more intuitive in the user's poit of view.

Reply:
fair enough

Reply:
@Guilherme2112 The optional arguments are not necessarily coupled with the command run.
You could do something like:

```ruby
def release(version_number, release_date = Date.today.strftime(Codelog::Config.date_input_format))
  if options[:preview]
    Codelog::Command::Preview.run version_number, release_date
  else
    Codelog::Command::Release.run version_number, release_date
  end
end
```